src/resources/Incorporations/unlimitedCompany.ts

```diff
@@ -53,36 +53,23 @@ export const UnlimitedCompanyResource: IncorporationResourceIF = {
   incorporationAgreement: {
     helpSection: [
       {
-        header: 'What is the sample Incorporation Agreement and Table 1 Articles?',
+        header: 'What is the sample Incorporation Agreement and Unlimited Liability Company Articles?',
         helpText: [
-          `The sample Incorporation Agreement and Table 1 Articles is a template that you can use to create an
-            incorporation agreement and articles for your company. It uses all the standard provisions suggested by
-            legislation.`,
-          `If you would like to customize any provisions in the Articles, you cannot use this sample. We recommend
-            seeking professional assistance from a lawyer or accountant to help you prepare your Articles.`
+          `The sample Incorporation Agreement and Company Articles is a template that you can use to create an 
+            incorporation agreement and articles for your company. It uses all the standard provisions by legislation. 
+            There are three types of samples depending on if you’re incorporating a Limited Company, a Benefit Company, 
+            or a Community Contribution Company. There is currently no Incorporation Sample for Unlimited 
+            Liability Companies.`,
+          `If you would like to customize any other provisions in the Articles, you cannot use these samples.
+            We recommend seeking professional assistance from a lawyer or accountant to help you prepare your articles.`
         ]
       },
       {
-        header: 'Can I use the sample Incorporation Agreement and Table 1 Articles?'
-      },
-      {
-        header: 'You can use the Incorporation Agreement and Table 1 Articles if:',
-        icon: 'mdi-check',
-        iconColor: 'green darken-2',
-        statements: [
-          `There are no special rights or restrictions attached to any class or series of shares in
-            the corporation's authorized share structure.`,
-          'You do not wish to change any of the standard provisions in the sample Articles.'
-        ]
-      },
-      {
-        header: 'You cannot use the Incorporation Agreement and Table 1 Articles if:',
-        icon: 'mdi-close',
-        iconColor: 'red',
-        statements: [
-          `There are special rights or restrictions attached to any class or series of shares in the corporation's
-            authorized share structure.`,
-          'You wish to change any of the standard provisions in the sample Articles.'
+        header: 'What is a Benefit Provision?',
```

Comment:
Do all of these filings display all of these help sections? If not then delete these sections from the resource files.

(Trying to avoid [Decoy Pattern](https://github.com/bcgov/lear/discussions/1825#discussioncomment-4278721).)

Reply:
Any update on this?

Reply:
Re-requested your review.